rpcserver.go

```diff
@@ -2294,17 +2322,55 @@ func marshalMintingBatch(batch *tapgarden.MintingBatch,
 			return nil, err
 		}
 
-		rpcBatch.Assets = append(rpcBatch.Assets, &mintrpc.MintAsset{
+		rpcAssets = append(rpcAssets, &mintrpc.MintAsset{
 			AssetType:    taprpc.AssetType(seedling.AssetType),
 			AssetVersion: assetVersion,
 			Name:         seedling.AssetName,
 			AssetMeta:    seedlingMeta,
 			Amount:       seedling.Amount,
 			GroupKey:     groupKeyBytes,
+			GroupAnchor:  groupAnchor,
```

Comment:
Ok, I see it being used now.

Reply:
Oops, that should've been in the first commit. Fixed.